modules/forms/client-react/FormError.ts

```diff
@@ -0,0 +1,41 @@
+import { isApolloError } from 'apollo-client';
+import { GraphQLError } from 'graphql';
+
+export const isFormError = (err: any) => err instanceof FormError;
+
+export class FormError {
+  public _errors: { [key: string]: any };
+
+  constructor(errorMsg: string, err?: any) {
+    if (!!err) {
```

Comment:
`if` statement automatically converts a value into `Boolean`, so there is no need to it explicitly.

Reply:
Fixed